augur/tasks/github/issues/tasks.py

```diff
@@ -129,19 +130,20 @@ def process_issues(issues, task_name, repo_id, logger) -> None:
 
         # insert contributors from these issues
         logger.info(f"{task_name}: Inserting {len(contributors)} contributors")
-        session.insert_data(contributors, Contributor, ["cntrb_login"])
+        session.insert_data(contributors, Contributor, ["cntrb_id"])
                             
 
         # insert the issues into the issues table. 
         # issue_urls are gloablly unique across github so we are using it to determine whether an issue we collected is already in the table
         # specified in issue_return_columns is the columns of data we want returned. This data will return in this form; {"issue_url": url, "issue_id": id}
         logger.info(f"{task_name}: Inserting {len(issue_dicts)} issues")
-        issue_natural_keys = ["repo_id", "gh_issue_id"]
+        issue_natural_keys = ["issue_url"]
```

Comment:
@sgoggins / @IsaacMilarky This should be `["repo_id", "gh_issue_id"]` not `["issue_url"]`. If a repo moves, the issue URL will change and we will get duplicate `gh_issue_ids` since the issues id stays the same, but the URL changes.

Reply:
@IsaacMilarky : What @ABrain7710 is saying here makes sense to me.